<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_main">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tasks">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_task_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new Task</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Task to work with</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pause">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Timer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_real_time">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;real_time*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_stats">
     <attribute name="title">
      <string>Stats</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Общее&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_all_time">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;time*&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
